osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/IControlPointVisualisationRedundant.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Beatmaps.ControlPoints;
+
+namespace osu.Game.Screens.Edit.Components.Timelines.Summary.Parts
+{
+    public interface IControlPointVisualisationRedundant
```

Comment:
Does this interface need to be coupled to the concept of redundancy via the name? Could it not just be named `IControlPointVisualisation`? I don't really like that "redundant" suffix, feels way too specific.

Reply:
Have made this change.